### PR TITLE
Norwegian translations

### PR DESCRIPTION
**Pull request by rickdijk**
Here you go, Norwegian translation just before the end of 2020! 😉

### CODE REVIEW

File: MVP/Resources/Translations.no.resx [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+        <value>Velg opptil 2 ytterlige teknologiområder som også kan gjelde for denne aktiviteten.</value>
+    </data>
+    <data name="contributionform_visibility_placeholder" xml:space="preserve">
+        <value>Velg aktivitetens synlighetsnivået</value>
+    </data>
+    <data name="setting_language_title" xml:space="preserve">
+        <value>Språk</value>
+    </data>
+    <data name="setting_language_description" xml:space="preserve">
+        <value>Som satdard velger vi språket som du bruker på enheten din hvis oversettelser ar tilgjengelig. Hvis du derimot ønsker å bruke denne appen på et annet språk, kan du overstyre det her.</value>
```
Comment (sthewissen) [inline MVP/Resources/Translations.no.resx:127]: Satdard should be standard?

File: MVP/Resources/Translations.no.resx [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+        <value>Lag direkte bidrag basert på en URL på klippetavla. Fyller ut alt mulig for deg, slik at du raskt kan legge til nye bidrag.</value>
+    </data>
+    <data name="acknowledgementtype_translator" xml:space="preserve">
+        <value>Oversetter</value>
+    </data>
+    <data name="acknowledgementtype_tester" xml:space="preserve">
+        <value>Tester</value>
+    </data>
+    <data name="about_credits_description" xml:space="preserve">
+        <value>Uten det råeste folket som er oppført nedenfor, denne appen kunne ikke ha vært mulig. Disse har derfor fortjent en plass her å feire deres bidrag.</value>
```
Comment (sthewissen) [inline MVP/Resources/Translations.no.resx:442]: Everywhere I look råeste seems to translate to raw? :D

File: MVP/Resources/Translations.no.resx [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+        <value>Slett bidrag</value>
+    </data>
+    <data name="about_title" xml:space="preserve">
+        <value>Om denne appen</value>
+    </data>
+    <data name="about_why_title" xml:space="preserve">
+        <value>Hvorfor ble denne appen bygget?</value>
+    </data>
+    <data name="about_why_description" xml:space="preserve">
+        <value>Det har alltid vært et behov for Microsoft MVPs å styre samfunnsaktivitene sine på en enklere måte. Nettportalen fungerer fint, men å ha noen lignende i lomma for å lage aktiviter i farta er helt rått. Som er hva dette prosjektet er for!</value>
```
Comment (sthewissen) [inline MVP/Resources/Translations.no.resx:97]: Ratt translates to raw everywhere I look, probably slang but I'd love to know :D

**Review by sthewissen — COMMENTED** [review]
I did my utmost to review, just because I can 😂 Please laugh at me in the comments.

Looks awesome though!

File: MVP/Resources/Translations.no.resx (hunk 2 above)
Comment (rickdijk) [reply MVP/Resources/Translations.no.resx:442]: Yes, literally that is what it means. However, it's used in speaking language as 'cool' or 'awesome'

File: MVP/Resources/Translations.no.resx (hunk 3 above)
Comment (rickdijk) [reply MVP/Resources/Translations.no.resx:97]: Yeah I'll update it to 'cooked' when the app is finished :D

File: MVP/Resources/Translations.no.resx (hunk 1 above)
Comment (rickdijk) [reply MVP/Resources/Translations.no.resx]: Whoops, fixed it

**Pull request merged**